protected/humhub/modules/admin/assets/AdminPendingRegistrationsAsset.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2017 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\admin\assets;
+
+use yii\web\AssetBundle;
+use yii\web\View;
+
+class AdminPendingRegistrationsAsset extends AssetBundle
+{
+    /**
+     * @inheritdoc
+     */
+    public $jsOptions = [
+        'position' => View::POS_END
+    ];
+    public $sourcePath = '@admin/resources';
+
+    /**
+     * @inheritdoc
+     */
+    public $js = [
+        'js/humhub.admin.pending.registrations.js'
+    ];
+
```

Comment:
You could register the url here e.g:

```
public static function register($view)
    {
        $view->registerJsConfig('admin.pending.registrations', [
            'deleteAllUrl' => Url::to('...')
        ]);

        return parent::register($view); // TODO: Change the autogenerated stub
    }
```

In you js module call:

```
var url = module.config.deleteAllUrl;
```

Reply:
Done with JsWidget